### PR TITLE
node: Call method without arguments or empty object

### PR DESCRIPTION
**Pull request by xiaozhenliu-gg5**
fixes#1893

### CODE REVIEW

**Comment by xiaozhenliu-gg5** [comment]
Thanks @alexander-fenster :) 
Yes, looking at the fail tests, and will make changes accordingly.

File: src/test/java/com/google/api/codegen/gapic/testdata/nodejs/nodejs_library.baseline [hunk 1]
```diff
@@ -5936,7 +5936,6 @@ class LibraryServiceClient {
    * });
    */
   streamShelves(request, options) {
-    request = request || {};
```
Comment (alexander-fenster) [inline src/test/java/com/google/api/codegen/gapic/testdata/nodejs/nodejs_library.baseline]: For server streaming we can still have it :)

Comment (xiaozhenliu-gg5) [reply src/test/java/com/google/api/codegen/gapic/testdata/nodejs/nodejs_library.baseline]: Got it, thanks.

Comment (xiaozhenliu-gg5) [reply src/test/java/com/google/api/codegen/gapic/testdata/nodejs/nodejs_library.baseline]: Before pulling new updates from master, all the tests passed, while now seems bazel_test starts to fail.

Comment (alexander-fenster) [reply src/test/java/com/google/api/codegen/gapic/testdata/nodejs/nodejs_library.baseline]: It's flaky, @vam-google maybe will fix it when he gets back from his vacation.

Comment (xiaozhenliu-gg5) [reply src/test/java/com/google/api/codegen/gapic/testdata/nodejs/nodejs_library.baseline]: Sounds cool, then we can hold this until the tests are fixed. Thanks

Comment (alexander-fenster) [reply src/test/java/com/google/api/codegen/gapic/testdata/nodejs/nodejs_library.baseline]: No. We just merge it if the bazel test is the only one that fails.

**Pull request merged**